lang/es-es.js

```diff
@@ -98,8 +98,8 @@ export default {
 	"components.selection.select-all": "Seleccionar todo",
 	"components.selection.select-all-items": "Seleccione los {count} elementos",
 	"components.selection.selected": "{count} seleccionados",
-	"components.switch.visible": "Visible", // mfv-translated
```

Comment:
Perhaps we should enhance the linter to check an "ignore file" for a list of terms to ignore for the "untranslated" rule. @bearfriend , thoughts?

Reply:
I think this is something we can follow up on separately.

Reply:
This was actually my original intention years ago. I wanted to associate translations with a hash of the source so if it changed anything "untranslated" would need to be manually marked as ignored again, that way `untranslated` issues would always be something to address, and could be errors instead of warnings.